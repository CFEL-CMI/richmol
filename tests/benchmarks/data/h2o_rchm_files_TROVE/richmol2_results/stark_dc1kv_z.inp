
tdse_init(t_start=0.0, t_end=0.0, dt=0.01, units="ps", solver="split_zhexpv", accur=1.0e-12, no_krylov=10)

fortran_tdse(flog="fortran.log")

h0 = States("../energies_j0_j40_MARVEL_HITRAN.rchm",{'emax':10000,'fmax':5,'fmin':0,'mmin':-5,'mmax':5,'dm':1,'df':1})
mu = Tensor("../matelem_MU_j<j1>_j<j2>.rchm", rank=1, ind_sym="none", units="Debye", me_thresh=1.0e-12)

edc = Field(form=["0","0","edc0"], params={"edc0":10000}, units="V/cm")

ham = -1.0*mu*edc

eigen_run(h0, ham, {"print_rate":1,"file":"stark_energies_10000","print_coef":"wavefunc"}, state_ind=range(0,1))

