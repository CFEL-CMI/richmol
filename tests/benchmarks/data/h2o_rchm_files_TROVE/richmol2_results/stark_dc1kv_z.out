
All output from the Fortran TDSE module will be printed into file: fortran.log


Initialize tdse module

   initial time: 0.0 ps
   terminal time: 0.0 ps
   time step: 0.01 ps

   Use Planck constant for time units ps and energy units cm^-1, hbar = 5.30883730500664

   Initialize grid of time-points
   Number of points: 1

   TDSE solver: split_zhexpv
   Requested accuracy of solution: 1e-12
   Number of Krylov vectors: 10


Initialize field-free basis

   States file: ../tmp/energies_j0_j40_MARVEL_HITRAN.rchm
   State filters: {'emax': 10000, 'fmax': 5, 'fmin': 0, 'mmin': -5, 'mmax': 5, 'dm': 1, 'df': 1}

   selected F quanta: [0, 1, 2, 3, 4, 5]

         F   no.levels
       0.0       23
       1.0       69
       2.0      113
       3.0      152
       4.0      189
       5.0      227

         F   no.levels (incl. degeneracies)
       0.0       23
       1.0       69
       2.0      113
       3.0      152
       4.0      189
       5.0      227

   selected M quanta: [-5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5]

   total number of basis functions: 6057

   Initialize Fortran TDSE module ...
   ... done


Add tensor operator

   Matrix element files template: ../tmp/matelem_MU_j<j1>_j<j2>.rchm

   Rank: 1
   Index symmetry: none
   Number of unique Cartesian components: 3
     1    x      1
     2    y      1
     3    z      1

   Scan matrix element files
       F1     F2 transposed?    filename
      0.0    0.0       False    ../tmp/matelem_MU_j0_j0.rchm
      0.0    1.0       False    ../tmp/matelem_MU_j0_j1.rchm
      1.0    0.0        True    ../tmp/matelem_MU_j0_j1.rchm
      1.0    1.0       False    ../tmp/matelem_MU_j1_j1.rchm
      1.0    2.0       False    ../tmp/matelem_MU_j1_j2.rchm
      2.0    1.0        True    ../tmp/matelem_MU_j1_j2.rchm
      2.0    2.0       False    ../tmp/matelem_MU_j2_j2.rchm
      2.0    3.0       False    ../tmp/matelem_MU_j2_j3.rchm
      3.0    2.0        True    ../tmp/matelem_MU_j2_j3.rchm
      3.0    3.0       False    ../tmp/matelem_MU_j3_j3.rchm
      3.0    4.0       False    ../tmp/matelem_MU_j3_j4.rchm
      4.0    3.0        True    ../tmp/matelem_MU_j3_j4.rchm
      4.0    4.0       False    ../tmp/matelem_MU_j4_j4.rchm
      4.0    5.0       False    ../tmp/matelem_MU_j4_j5.rchm
      5.0    4.0        True    ../tmp/matelem_MU_j4_j5.rchm
      5.0    5.0       False    ../tmp/matelem_MU_j5_j5.rchm

   Tensor name: MU
   Number of irreps: 1
   Number of unique Cartesian components: 3
   Convert electric field tensor from debye to AU using factor: 0.393456

   Add new tensor in Fortran TDSE module ...
   Threshold for neglecting mtrix elements: 1e-12
   ... done

   Current tensor index: 1


Add electric field

   Number of time-points: 1

   Initialize field from input formula: ['0', '0', 'edc0']
   Parameters: {'edc0': 10000}

   Convert electric field from v/cm to V/m using factor: 100.0
Store stationary state no. 0 at a time = 0.0 into file wavefunc.0

Initial wavepacket

   Total number of basis functions: 6057
   Number of initial coefficients: 20

   #basis func    M    F   ID ideg   coefficient
          2642    0    0    1    1   9.99988316e-01   0.00000000e+00i
          2643    0    0    2    1   4.29011598e-08  -4.91667494e-28i
          2672    0    1    2    1  -9.97375513e-32  -4.83394343e-03i
          2673    0    1    5    1  -1.19790296e-30  -7.60085440e-06i
          2674    0    1    8    1  -5.31241969e-28   2.32911174e-07i
          2675    0    1   11    1  -2.31276587e-27  -3.56133569e-07i
          2676    0    1   13    1   4.49457335e-28  -1.63440628e-06i
          2677    0    1   17    1  -1.73228365e-27  -1.16425728e-08i
          2678    0    1   20    1   2.86320410e-28  -4.23174442e-08i
          2679    0    1   22    1  -3.94678802e-29  -5.19748317e-07i
          2681    0    1   29    1   3.52755302e-27  -1.09077072e-08i
          2682    0    1   31    1  -3.78673050e-28   4.54442470e-08i
          2683    0    1   35    1   4.62584119e-27   6.04258788e-08i
          2684    0    1   37    1   9.08574069e-28  -1.63635721e-07i
          2685    0    1   41    1  -4.09287909e-27   2.19886283e-08i
          2690    0    1   55    1   4.42163324e-29   5.44357476e-08i
          2734    0    2    1    1  -6.81485425e-06   6.25148886e-28i
          2735    0    2    5    1  -4.50015045e-06  -6.05261650e-28i
          2736    0    2    6    1   2.10725070e-08   5.64434211e-27i
          2737    0    2   10    1   2.19626833e-08   1.12264475e-26i

   Normalize coefficient vector by 1/sqrt( 0.9999999999999998 )
